R/make_standata.R

```diff
@@ -290,6 +290,11 @@ standata_basis_ac <- function(x, data, ...) {
       out$locations <- NA
     }
   }
+  if (has_ac_class(x, "arma")) {
```

Comment:
you are overwriting the complete "out" variable instead of adding to it here. Please add the new variables to the existing list.